packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerCorrectness.test.ts

```diff
@@ -181,12 +181,16 @@ export function testCorrectness() {
 			describe("Trunk eviction", () => {
 				it("Evicts trunk commits according to a provided minimum sequence number", () => {
 					const { manager } = testChangeEditManagerFactory({});
+					const trimmedCommits = new Set();
+					const expectedTrimmedRevisions = new Set<RevisionTag>();
+					manager.localBranch.on("branchTrimmed", (trimmedRevisions) => {
```

Comment:
What do you think about making a helper `trackTrimmed(branch): ReadonlySet<RevisionTag>` to get rid of the four lines of boilerplate in all these tests?